Modules/Client/Services/ClientService.php

```diff
@@ -147,7 +156,14 @@ public function store($data)
         $data['status'] = 'active';
         $data['client_id'] = Client::max('client_id') + 1;
 
-        return Client::create($data);
+        $store = Client::create($data);
```

Comment:
Its good practice to use meaningful variable name. 
```suggestion
        $newClient = Client::create($data);
```